src/nni_manager/config/pytorchjob-crd-v1alpha2.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
how about putting these json files under training_service/kubeflow?

Reply:
Sure. Will move

Reply:
Done. Thanks for the comments.